lib/hammer_cli_foreman_openscap/arf_report.rb

```diff
@@ -17,9 +25,15 @@ class ListCommand < HammerCLIForeman::ListCommand
 
     class InfoCommand < HammerCLIForemanOpenscap::InfoCommand
       output ListCommand.output_definition do
-        field :host_id, _("Host Id")
-        field :openscap_proxy_id, _("Openscap proxy Id")
-        field :openscap_proxy_name, _("Openscap proxy name")
+        from :host do
+          field :id, _("Host Id")
+        end
+        from :openscap_proxy do
+          field :id, _("Openscap proxy Id")
```

Comment:
if this plugin is used against older version of foreman_openscap, we shouldn't print it, @tstrachota correct me if I'm wrong, but we can use `:hide_blank => true` which is what we did for [subnet description in core](https://github.com/theforeman/hammer-cli-foreman/blob/master/lib/hammer_cli_foreman/subnet.rb#L24)

if you agree I think we should add it for other fields and ListCommand too